scripts/ci/run-compile-tests.sh

```diff
@@ -15,8 +15,12 @@ if [ $(echo -n $d | wc -l) -ne 0 ] ; then
     exit 1
 fi
 
+# prepare collectd build
+rm -rf /tmp/collectd
+git clone https://github.com/collectd/collectd.git /tmp/collectd
```

Comment:
you are using collected master branch, better to set to a specific tag

Reply:
It is a CI job, I think it is fine to detect as soon as possible if something changed that break the plugin.

Reply:
ok